packages/browser/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -197,7 +192,6 @@ export class SessionInfoManager implements ISessionInfoManager {
       webId,
       isLoggedIn: isLoggedIn === "true",
       redirectUrl,
-      idToken,
       refreshToken,
```

Comment:
Should we be validating that the refreshToken is a JWT here? (I assume it is)

Reply:
It isn't necessarily: the refresh token, like the access token, is opaque to the client, it's an implementation detail of the OIDC provider.